data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Crown Copyright
+ * Copyright 2020 Crown Copyright
```

Comment:
revert, there are no other changes to this file

Reply:
removed